lib/output.js

```diff
@@ -1456,7 +1456,7 @@ function OutputStream(options) {
             parent = output.parent(level++);
             if (parent instanceof AST_Call && parent.expression === node) return;
         } while (parent instanceof AST_PropAccess && parent.expression === node);
-        output.print("/*" + self.pure + "*/");
+        output.print(typeof self.pure == "string" ? "/*" + self.pure + "*/" : "/*@__PURE__*/");
```

Comment:
> I am invoking the hypothetical user request on behalf of myself.

Okay, you win 😈

Reply:
And thanks for correcting my one-liner patch - you knew what I meant.